src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/ISimdVector_2.cs

```diff
@@ -0,0 +1,450 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Numerics;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace System.Runtime.Intrinsics
+{
+    /// <summary>Defines a single instruction, multiple data (SIMD) vector type.</summary>
+    /// <typeparam name="TSelf">The type that implements the interface.</typeparam>
+    /// <typeparam name="T">The type of the elements in the vector.</typeparam>
+    internal interface ISimdVector<TSelf, T>
+        : IAdditionOperators<TSelf, TSelf, TSelf>,
+          // IAdditiveIdentity<TSelf, TSelf>,
+          IBitwiseOperators<TSelf, TSelf, TSelf>,
+          // IComparisonOperators<TSelf, TSelf, bool>,
+          // IDecrementOperators<TSelf>,
+          IDivisionOperators<TSelf, TSelf, TSelf>,
+          IEqualityOperators<TSelf, TSelf, bool>,
+          IEquatable<TSelf>,
+          // IIncrementOperators<TSelf>,
+          // IMinMaxValue<TSelf>,
+          // IModulusOperators<TSelf, TSelf, TSelf>,
+          // IMultiplicativeIdentity<TSelf, TSelf>,
+          IMultiplyOperators<TSelf, TSelf, TSelf>,
+          // IShiftOperators<TSelf, int, TSelf>
+          // ISpanFormattable,
+          ISubtractionOperators<TSelf, TSelf, TSelf>,
+          IUnaryNegationOperators<TSelf, TSelf>,
+          IUnaryPlusOperators<TSelf, TSelf>
```

Comment:
The commented out APIs would expose "more" than what the `Vector64/128/256<T>` types already do. We should likely explore whether those APIs make sense (I expect in many cases they will).

For cases like `IAdditionOperators` this adds `operator checked +` and doesn't give it a "correct" implementation. We could do so and it may even be desirable to do that longer term.